stable/insights-agent/templates/_uploader.yaml

```diff
@@ -48,9 +48,16 @@
   {{- toYaml . | nindent 2 }}
   {{- end }}
   {{ include "proxy-env-spec" . | indent 2 | trim }}
+  {{- if and .Values.global.sslCertFile.secretName .Values.global.sslCertFile.secretKey }}
+  {{ include "ssl-cert-file-env-spec" . | indent 2 | trim }}
+  {{- end }}
+
   volumeMounts:
   - name: output
     mountPath: /output
+{{- if and .Values.global.sslCertFile.secretName .Values.global.sslCertFile.secretKey }}
```

Comment:
it looks like this `if` and the one above are duplicated in side the `ssl-cert-file-volumemount-spec` definition

Reply:
Ah yes - thanks, I missed that one after I moved the if inside of the defines.